webui/src/pages/auth/groups/index.tsx

```diff
@@ -93,6 +94,8 @@ const GroupsContainer = () => {
 
     if (error) return <Error error={error}/>;
     if (loading) return <Loading/>;
+    const {RBAC: rbac} = useLoginConfigContext();
+    const headers = rbac !== 'simplified' ? ['', 'Group ID', 'Created At']:['', 'Group ID', 'Permission', 'Created At'] ;
```

Comment:
Nit: Add spaces, don't use a reversed comparison.
```suggestion
    const headers = rbac === 'simplified' ? ['', 'Group ID', 'Permission', 'Created At'] : ['', 'Group ID', 'Created At'];
```
and perhaps space it out over some lines.